web/landing/src/Flow/Website/Controller/DocumentationController.php

```diff
@@ -20,6 +20,35 @@ public function __construct(
     ) {
     }
 
+    #[Route('/documentation/api/{page}', name: 'documentation_api', requirements: ['page' => '.*'])]
+    public function apiPage(string $page) : Response
```

Comment:
this one is only for development, it's not being dumped to a static html build but without it we won't be able to see those api references static html's during development